mimesis/decorators.py

```diff
@@ -37,16 +40,16 @@ def wrapper(*args, **kwargs):
                     'Locale {0} is not supported yet.'.format(locale),
                 )
             result = func(*args, **kwargs)
-            txt = ''.join([alphabet[i] for i in result if i in alphabet])
+            txt = ''.join(cast(Iterable, (alphabet[i] for i in result if i in alphabet)))
```

Comment:
I'm having a problem with `join` here, so this isn't yet in a working condition; e.g., I shouldn't have tried to `cast` this value to `Iterable`, it's just meant to show that `join` is complaining about requiring an `Iterable` type. Per `help(''.join)`:

> join(...) method of builtins.str instance
    S.join(iterable) -> str